webapp/src/components/data_selectors/jira_commentvisibility_selector/jira_commentvisibility_selector.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {ReactSelectOption} from 'types/model';
+
+import BackendSelector, {Props as BackendSelectorProps} from '../backend_selector';
+
+const stripHTML = (text: string) => {
+    if (!text) {
+        return text;
+    }
+
+    const doc = new DOMParser().parseFromString(text, 'text/html');
+    return doc.body.textContent || '';
+};
+
+type Props = BackendSelectorProps & {
+    searchCommentVisibilityFields: (params: {fieldValue: string}) => (
+        Promise<{data: {groups: {items: {name: string}[]}}; error?: Error}>
+    );
+    fieldName: string;
+};
+
+const JiraCommentVisibilitySelector = (props: Props) => {
+    const {value, isMulti, instanceID, searchCommentVisibilityFields} = props;
+    const fetchInitialSelectedValues = async (): Promise<ReactSelectOption[]> =>
+        ((!value || (isMulti && !value.length)) ? [] : commentVisibilityFields(''));
+
+    const commentVisibilityFields = async (inputValue: string): Promise<ReactSelectOption[]> => {
+        const params = {
+            fieldValue: inputValue,
+            instance_id: instanceID,
+            expand: 'groups',
```

Comment:
If this is always set to `groups` for this endpoint, then it seems like we can omit `expand` from the API request, and have the server just always use that value when fetching the visibilities from Jira. Otherwise we unnecessarily allow arbitrary expands for the endpoint.